src/components/content-tab-import-export.tsx

```diff
@@ -162,11 +176,13 @@ const ImportSite = ( props: { selectedSite: SiteDetails } ) => {
 	};
 
 	const startLoadingCursorClassName =
-		loadingServer[ props.selectedSite.id ] && 'animate-pulse duration-100 cursor-wait';
+		loadingServer[ props.selectedSite.id ] &&
+		! isSyncing &&
+		'animate-pulse duration-100 cursor-wait';
 
 	const isImporting = currentProgress?.progress < 100 && ! isAnySitePulling && ! isAnySitePushing;
-	const isImported = currentProgress?.progress === 100 && ! isDraggingOver;
-	const isInitial = ! isImporting && ! isImported;
+	const isImported = currentProgress?.progress === 100 && ! isDraggingOver && ! isSyncing;
+	const isInitial = ( ! isImporting && ! isImported ) || isSyncing;
```

Comment:
In which cases we need to add `isSyncing` ? I removed the variable and I got the same results.

```suggestion
	const isInitial = ! isImporting && ! isImported;
```

Reply:
You are right! This is no longer necessary.

I have originally added the check before I set the `'animate-pulse duration-100 cursor-wait'` conditionally based on `isSyncing` above. I did not realize we don't need the check here anymore. Committing your suggestion. Thanks! 🙂